libraries/AP_HAL_ChibiOS/hwdef/BlitzF745AIO/hwdef-bl.dat

```diff
@@ -38,5 +38,11 @@ PA15 FLASH1_CS CS
 PE4 OSD1_CS CS
 PA4 GYRO1_CS CS
 
+# Keep UART RX pins high to prevent ELRS going into bootloader mode
+PA2 USART2_TX USART2 OUTPUT HIGH
+PA3 USART2_RX USART2 OUTPUT HIGH
+PB10 USART3_TX USART3 OUTPUT HIGH
+PB11 USART3_RX USART3 OUTPUT HIGH
+
```

Comment:
**Tested with RX only - works fine**

```
# Keep UART RX pins high to prevent ELRS going into bootloader mode
PA1 UART4_RX UART4 OUTPUT HIGH
PA3 USART2_RX USART2 OUTPUT HIGH
PB11 USART3_RX USART3 OUTPUT HIGH
PE7 UART7_RX UART7 OUTPUT HIGH

# default to all pins low to avoid ESD issues
DEFAULTGPIO OUTPUT LOW PULLDOWN
```

Reply:
Thanks I added all the RX pins